Makefile

```diff
@@ -3,8 +3,8 @@ NAMESPACE=integreatly
 PROJECT=integreatly-operator
 REG=quay.io
 SHELL=/bin/bash
-PREVIOUS_TAG=1.13.2
-TAG=1.14.0
+PREVIOUS_TAG=1.14.0
+TAG=1.14.1
```

Comment:
we don't do bumps on each PR anymore

Reply:
@philbrookes I have reverted the changes to bump the version.